storage/shard.go

```diff
@@ -107,13 +108,13 @@ func (s *dbShard) forEachSeries(continueOnError bool, seriesFn databaseSeriesFn)
 	s.RUnlock()
 
 	// TODO(xichen): pool or cache this.
-	curSeries := make([]databaseSeries, 0, batchSize)
+	curEntries := make([]*dbShardEntry, 0, batchSize)
```

Comment:
Could we call this `currEntries` for consistency? I use `currThing` instead of `curThing` very frequently.  We could also just do a grep and see which one is more common and apply that, I don't mind either way but would like to be consistent.